### PR TITLE
Bug 1269966 - [TV][Home] There is a chance that cards remain in the m…

### PR DESCRIPTION
**Pull request by luke-chang**
…ain list after joining a folder

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ce1fd5f135ed041a0abc9165393b3cf3939bca12

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cd9e8e7107ee45d632cecaa910fcdfafebbbb32d

File: shared/js/smart-screen/card_manager.js [hunk 1]
```diff
@@ -33,6 +33,8 @@
 
     _asyncSemaphore: undefined,
```
Comment (rexboy7) [inline shared/js/smart-screen/card_manager.js]: I'd prefer some comments here for it's not a clean solution :-/

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8944b927153017c2573c36dc21afb58378bf3413

**Pull request merged**